frontend/app/templates/projects.hbs

```diff
@@ -0,0 +1 @@
+{{outlet}}
```

Comment:
This shouldn't be necessary. I believe this is what the default template would do.